src/site/shared/containers/ItemNav/index.scss

```diff
@@ -43,7 +43,7 @@ $tab-padding: 8px;
         font-weight: bold;
 
         width: calc(49% - 5px);
-        padding: 1vh 1vh 1vh 1vh;
+        padding: 10px 10px 10px 10px;
```

Comment:
We can condense this to just `padding: 10px`

Reply:
Fixed!